demos/projects/ESPRESSIF/aziotkit/components/azure-iot-kit-sensors/src/sensor_manager.h

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef SENSOR_MANAGER_H
+#define SENSOR_MANAGER_H
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+    typedef void (*button_cb)(void *);
+
+    void initialize_sensors();
+    float get_temperature();
+    float get_humidity();
+    float get_ambientLight();
+    void get_pitch_roll_accel(int *pitch, int *roll, int *accelX, int *accelY, int *accelZ);
+    void get_pressure_altitude(float *pressure, float *altitude);
+    void get_magnetometer(int *magnetometerX, int *magnetometerY, int *magnetometerZ);
+    bool check_for_shake(int accelX, int accelY, int accelZ);
+    void oled_clean_screen();
+    void oled_show_message( const uint8_t * pucMessage, uint32_t ulMessageLength );
+    void stop_motor();
+    void start_motor_with_speed(float speed);

```

Comment:
what are we using it for? I do not see any more pheripher added with this board. Is this for PWM?